### ISSUE TITLE
E3001 Invalid or unsupported Type AWS::GuardDuty::Master in eu-west-3

### ISSUE THREAD

**Issue by calebplum** [aws-cloudformation/cfn-python-lint#1839]
*cfn-lint version: cfn-lint 0.44.0*

*Description of issue.*
Error `E3001 Invalid or unsupported Type AWS::GuardDuty::Master in eu-west-3` when deploying a template containing an `AWS::GuardDuty::Master` resource in the `eu-west-3` region.

Please provide as much information as possible:
* Template linting issues:
  * Please provide a CloudFormation sample that generated the issue.
```
---

AWSTemplateFormatVersion: 2010-09-09
Description: Sets up a GuardDuty member account and accepts an invitation from a GuardDuty master account.

Resources:
  Detector:
    Type: AWS::GuardDuty::Detector
    Properties:
      Enable: True

  GuardDutyMaster:
    Type: AWS::GuardDuty::Master
    Properties:
      DetectorId: !Ref Detector
      InvitationId: {master_invitation_id}
      MasterId: '{master_account_number}'
```

  * If present, please add links to the (official) documentation for clarification.
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-guardduty-master.html
  * Validate if the issue still exists with the latest version of `cfn-lint` and/or the latest Spec files
Confirmed that issue persists after updating cfn-lint to 0.44.0 and running `cfn-lint --update-specs`. Confirmed template deploys successfully without linting.

**Comment by PatMyron** [comment]
**Can workaround with [resource-level ignores](https://github.com/aws-cloudformation/cfn-python-lint#resource-based-metadata) until this fix is released:**

```yaml
  Resource:
    Type: AWS::GuardDuty::Master
    Metadata:
      cfn-lint:
        config:
          ignore_checks:
          - E3001
```
